src/extension.ts

```diff
@@ -1,129 +1,107 @@
 import * as vscode from "vscode";
 
-const TRANSPARENT_COLORS: { [key: string]: string } = {
-  "editorInfo.border": "#00000000",
-  "editorInfo.background": "#00000000",
-  "editorInfo.foreground": "#00000000",
-  "editorError.border": "#00000000",
-  "editorError.background": "#00000000",
-  "editorError.foreground": "#00000000",
-  "editorWarning.border": "#00000000",
-  "editorWarning.background": "#00000000",
-  "editorWarning.foreground": "#00000000",
-};
+const TRANSPARENT_COLOR = "#00000000";
+const SQUIGGLE_TYPES = ["Error", "Info", "Warning"] as const;
```

Comment:
```suggestion
const SQUIGGLE_TYPES = ["Error", "Hint", "Info", "Warning"] as const;
```

Any reason we shouldn't just support Hint squiggles too?

Reply:
Might as well, just don't know if it makes any difference now!